sister/word_embedders.py

```diff
@@ -30,7 +30,7 @@ def get_word2vec(lang: str = "en"):
     # Download.
     urls = {
             "en": "https://s3.amazonaws.com/dl4j-distribution/GoogleNews-vectors-negative300.bin.gz",
-            "ja": "http://public.shiroyagi.s3.amazonaws.com/latest-ja-word2vec-gensim-model.zip"
+            "hi": "http://public.shiroyagi.s3.amazonaws.com/latest-hi-word2vec-gensim-model.zip"
```

Comment:
As same as above, please not remove line for "ja" :)

and I think this URL for word2vec for "hi" is invalid.
This location is Japanese specific.

If you know valid URL provides word2vec model for "hi" please use it instead.